bokeh/server/configure.py

```diff
@@ -88,7 +88,7 @@ def configure_flask(config_argparse=None, config_file=None, config_dict=None):
 def register_blueprint():
     global registered
     if registered:
-        warnings.warn(
+        logging.warn(
```

Comment:
This is the only bit I am a bit unsure of. I think using logging is better, since that will pass through our own logging system. But maybe there was a reason to use `warnings` instead.